docs/rfc/ethereum_attestation.md

```diff
@@ -0,0 +1,126 @@
+# RFC: Ethereum attestation
+
+* Author: @CodeSandwich
+* Date: 2021-03-05
+* Status: draft
+* Community discussion: n/a
+
+## Motivation
+
+The attestation between Link and Ethereum is a valuable building block for a user identity.
+It brings the Link reputation coming from projects and contributions to the
+Ethereum world of DAOs and donations, where it's important to know, who's behind an address.
+On the other hand it lends Ethereum account reputation with its assets and undeniable history
+to Link to build user's trust in the identity.
+
+## Overview
+
+This RFC is built on top of [Identity Proofs RFC][rfc].
+It introduces support for Ethereum address claims on Link
+and a smart contract on Ethereum to make Link identity claims on Ethereum.
+
+## Link identity JSON extension
+
+The identity JSON supports a new key: `ethereum`.
+Under this key there is stored an ethereum address claim following this convention:
+
+- `account` - the claimed ethereum address, encoded according to [EIP-55][eip-55],
+e.g. using [ethers.js][ethers-addr]
+- `proof` - always `null`
+
+Example:
+```json
+{
+    "ethereum": {
+        "account": "0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B",
+        "expiration": {
+            "created": 1614768373,
+            "expires": 31536000
+        },
+        "proof": null
+    }
+}
+```
+
+## Ethereum smart contract
+
+A new smart contract is deployed to the network, which is used to publish link claims:
+
+```solidity
+contract Claims {
+    event Claimed(address indexed addr);
+    function claim(uint8 version, uint256 id) public;
+}
+```
+
+In order to claim an ID, call `claim` using your Ethereum account.
+It will emit an event `Claimed`, which later can be queried to discover your attestation.
+The claims have no expiration date and don't need to be renewed.
+
+Every new claim invalidates previous ones made with the same account.
+To only revoke a claim without creating a new one, use ID `0`,
+which is guaranteed to not match any existing identity.
+
+Currently supported `version` values:
+- `1` - an `id` is a SHA-1 root hash. The excess high bytes are zeros, e.g. for hash
+`fb3102b74d7254eed7f18a31a3ba1ea946bb1a99` the passed `id` is
+`000000000000000000000000fb3102b74d7254eed7f18a31a3ba1ea946bb1a99`
+- `2` - an `id` is a SHA-256 root hash
+
+Everybody must use the same smart contract address.
+Claims sent to an address not known by the users won't be noticed by anyone and will be useless.
```

Comment:
> an address not known by the users won't be noticed by anyone and will be useless.

Could rephrase to in a way so that it transmits the idea of there being an official contract address that is the only one taken into account?

Reply:
Fixed